### PR TITLE
Admin sponsors

### PR DESCRIPTION
**Pull request by KaiWissel**

### CODE REVIEW

File: client/src/types/Sponsor.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export interface Sponsor {
+  id?: string;
+  name: string;
+  website: string;
+  tagline: string;
+  isGoldSponsor: boolean;
+  sponsorInSeasons: number[];
+  contacts: Contact;
```
Comment (JEK58) [inline client/src/types/Sponsor.ts:8]: Warum ist das Mehrzahl, wenn nur ein Kontakt erlaubt ist?

**Review by JEK58 — COMMENTED** [review]
After upload of a sponsor logo the cache needs to be emptied. At the moment the old logo still is served.

What happens if a huge image is uploaded? Right now we will serve that to the user even if it's 10MB. There should either be an upload limit, or image processing that generates versions like we did with flight photos.

Screenshot 2022-09-18 at 19 18 59

The URL 🧐
Either crop it or we do not allow such long URLs or the cell gets a max width.

Screenshot 2022-09-18 at 19 19 42

UI in general is not so nice. It's not a public page but still. 😜

File: client/src/types/Sponsor.ts (hunk 1 above)
Comment (KaiWissel) [reply client/src/types/Sponsor.ts:8]: Bei Clubs ist es wirklich Mehrzahl; Wurde halt so aus dem alten Cup übernommen.

**Comment by KaiWissel** [comment]
> After upload of a sponsor logo the cache needs to be emptied. At the moment the old logo still is served.
> 
> What happens if a huge image is uploaded? Right now we will serve that to the user even if it's 10MB. There should either be an upload limit, or image processing that generates versions like we did with flight photos.
> 
> Screenshot 2022-09-18 at 19 18 59
> 
> The URL 🧐 Either crop it or we do not allow such long URLs or the cell gets a max width.
> 
> Screenshot 2022-09-18 at 19 19 42
> 
> UI in general is not so nice. It's not a public page but still. 😜

Also das Backend erstellt wie bei den FlightPhotos auch kleinere Kopien der Originalvariante

**Comment by KaiWissel** [comment]
@JEK58 bump

**Comment by JEK58** [comment]
Cache wird allerdings noch immer nicht gelöscht nach hochladen eines neuen Logos.

**Comment by KaiWissel** [comment]
@JEK58 Cache wird jetzt auch gelöscht

**Review by JEK58 — APPROVED** [review]

**Pull request merged**